src/main/java/br/edu/ufcg/computacao/alumni/core/MatchesFinder.java

```diff
@@ -95,18 +99,24 @@ public static synchronized MatchesFinder getInstance() {
 	private String[] normalize(String[] rawName) {
 		String[] normalizedName = new String[rawName.length];
 		for (int i = 0; i < rawName.length; i++) {
-			normalizedName[i] = (rawName[i].trim()).toUpperCase();
+			String str = rawName[i].trim().toUpperCase();
+			Normalizer.normalize(str, Normalizer.Form.NFD).replaceAll("[^\\p{ASCII}]", "");
+			normalizedName[i] = str;
 		}
 		return normalizedName;
 	}
 
-	private int compareNames(String[] namesToCompare, String[] names) {
+	private int compareNames(String[] namesToCompare, String[] names, int base) {
+		if (Arrays.asList(namesToCompare).isEmpty() && Arrays.asList(names).isEmpty()) {
+			return 30;
+		}
```

Comment:
Não entendi essa lógica. Em primeiro lugar, existem situações em que a condição desse if será avaliada como verdadeira? Até onde eu entendo, namesToCompare vem do nome do alumni (alumniParsedName) e nunca é vazio. Mesmo considerando que eu possa entrar no if, não entendo por que retornar 30 no caso dos nomes serem vazios. Intuitivamente eu retornaria 0. Se for pra retornar algo diferente de zero, eu retornaria 1.